.github/workflows/dependency-update.yaml

```diff
@@ -25,8 +25,34 @@ jobs:
       - name: Create Pull Request
         uses: peter-evans/create-pull-request@v3
         with:
-          title: "Updated npm dependencies w${{ steps.week.outputs.NUMBER }}"
-          branch: "dependencies/npm-updates-w${{ steps.week.outputs.NUMBER }}"
+          title: "Updated designer npm dependencies w${{ steps.week.outputs.NUMBER }}"
+          branch: "dependencies/designer-npm-updates-w${{ steps.week.outputs.NUMBER }}"
+          body: ${{ steps.update.outputs.body }}
+          labels: dependencies
+
+  appfrontend_npm_update_sync:
+    runs-on: ubuntu-latest
+    name: Update App frontend npm packages and sync package.json
+    steps:
+      - name: checkout code
+        uses: actions/checkout@v2
+
+      - name: Update package.json
+        id: update
+        uses: tjololo/npm-package-update-sync@v1
```

Comment:
Did we add functionality for bumping the minor version for `src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/package.json`.  if this package has dependency updates? 

If not we should specify this as a manual routine until we are able to automate this

Reply:
Currently it's not automated so we should add this to the manual routine.